### ISSUE TITLE
Llama-2 implementations on Cerebras

### ISSUE THREAD

**Issue by RicardoZhy1215** [Cerebras/modelzoo#53]
Hi, could you provide Llama-2 implementations on Cerebra?

**Comment by bhargav-cerebras** [comment]
these should be a good start: https://github.com/Cerebras/modelzoo/tree/main/src/cerebras/modelzoo/models/nlp/llama#llama-2 ?

**Comment by RicardoZhy1215** [comment]
> these should be a good start: https://github.com/Cerebras/modelzoo/tree/main/src/cerebras/modelzoo/models/nlp/llama#llama-2 ?

Yeah, I  think it is a good start, but I found that when I train a Llama model, it uses GPT2 model as the base model,and use different configs to get the Llama model?
